centreon/src/Centreon/Domain/Repository/NagiosServerRepository.php

```diff
@@ -48,72 +93,113 @@ public function getPaginationList($filters = null, int $limit = null, int $offse
     {
         $collector = new StatementCollector;
 
-        $sql = 'SELECT SQL_CALC_FOUND_ROWS * '
-            . 'FROM `' . $this->getClassMetadata()->getTableName() . '`';
+        $sql = 'SELECT SQL_CALC_FOUND_ROWS * FROM `:db`.`nagios_server`';
 
         if ($filters !== null) {
             $isWhere = false;
 
-            if (!empty($filters['search'])) {
-                $sql .= ' WHERE `' . $this->getClassMetadata()->getColumn('name') . '` LIKE :search';
+            if (! empty($filters['search'])) {
+                $sql .= ' WHERE `name` LIKE :search';
```

Comment:
Consistency with other modifications of this type.
```suggestion
            if ($filters['search'] ?? false) {
                $sql .= ' WHERE `name` LIKE :search';
```